newIDE/app/src/SceneEditor/index.js

```diff
@@ -598,24 +598,19 @@ export default class SceneEditor extends React.Component<Props, State> {
     const { project, layout } = this.props;
 
     //eslint-disable-next-line
-    const answer = confirm(
-      'Do you want to remove all references to this object in groups and events (actions and conditions using the object)?'
-    );
```

Comment:
Have you considered the impact of removing this? 😬

It seems that you've entirely investigated what this feature is and how it is useful to users. I would recommend spend more time investigating and being sure about what your changes are implying.

Reply:
I realised the impact moments after I pushed the code. The prompt clearly warns about the same. It's a misjudgement from my side. I shall revert all the undesired changes.

Please correct me if I'm wrong, but I noticed that the statement `if(!answer) return;` was missing soon after this statement, meaning that the object would be deleted regardless of the response by the user. I observed this during debugging.

Reply:
> Please correct me if I'm wrong, but I noticed that the statement if(!answer) return; was missing soon after this statement, meaning that the object would be deleted regardless of the response by the user. I observed this during debugging

Yes, because the question asked to the user is about if events should be removed or not. The object will be deleted in any case (ideally, we should show "yes/no" rather than "ok/cancel", i admit it's not super clear).